.dir-locals.el

```diff
@@ -0,0 +1,8 @@
+;;; Directory Local Variables
+;;; For more information see (info "(emacs) Directory Variables")
+
+((nil
+  (haskell-mode-stylish-haskell-path . "stylish-haskell")
+  )
+ (haskell-mode
+  (haskell-stylish-on-save . t)))
```

Comment:
do we really want an editor specific file ?

Reply:
That's a good question. By having this here we make sure that Emacs users don't have to setup anything extra (and if they don't use `stylish-haskell` for other projects, they don't have to set this up locally).

Alternatively I could put this in the README, but then everyone who uses Emacs would have to copy and paste this.

I don't know what's more sensible ...

Reply:
People might have their own `.dir-locals.el` file and get a conflict with this. The might also use a different way to setup stuff. But in reality it probably doesn't matter. Let's do it this way and see whether someone complains.

Reply:
Just replying here as well for the record: the reasons I went for including `.dir-locals.el` into our git repo are:

-  if people need a custom `.dir-locals.el` to edit files in this project then I think there's a high chance that other developers might want to use those settings too.
- by having a versioned `.dir-locals.el` everybody that uses Emacs don't need additional setup.